README.md

```diff
@@ -20,6 +20,7 @@ magenboy [path_to_rom] [other_optional_flags]
 ### Optional flags
 * `--log` - Print logs in debug mode to a file
 * `--file-audio` - Saves the audio to a file
+* `--full-screen` - Display the screen all over the screen
```

Comment:
Find a way to say it better its sounds like when the screen is full screen the form is in the middle.

Maybe boot to full screen mode?